kano_settings/set_style.py

```diff
@@ -39,16 +38,33 @@ def __init__(self, win):
         reset_widget = SetResetDesktop(self.win)
 
         wallpaper_label = Gtk.Label('BACKGROUND')
+        wallpaper_label_ebox = Gtk.EventBox()
+        wallpaper_label_ebox.add(wallpaper_label)
+        wallpaper_label_ebox.connect("realize", self._set_cursor_to_hand_cb)
+        wallpaper_label_ebox.show_all()
```

Comment:
I don't think these aren't needed, the self.win.show_all() at the end should recursively show these.